src/mesido/esdl/esdl_model_base.py

```diff
@@ -388,6 +475,19 @@ def _esdl_convert(self, converter: _AssetToComponentBase, assets: Dict, prefix:
                     conn = (port.id, connected_to.id)
                     if conn in connections or tuple(reversed(conn)) in connections:
                         continue
-
-                    self.connect(port_map[port.id], port_map[connected_to.id])
+                    if (
+                        asset.asset_type == "Pipe"
+                        or asset.asset_type == "ElectricityCable"
+                        or assets[
+                            name_to_id_map[port_map[connected_to.id].name.split(".")[0]]
+                        ].asset_type
```

Comment:
maybe add a comment on the asset type check in the ports, to make it easier understand when logical links should be created or normal connections.

Reply:
done